build-system/tasks/storybook/index.js

```diff
@@ -119,7 +119,8 @@ storybook.description =
   'Set up isolated development and testing for AMP components';
 
 storybook.flags = {
-  'build': 'Build a static web application (see https://storybook.js.org/docs)',
+  'build':
+    'Build a static web application (see https://storybook.js.org/docs). This outputs to examples/storybook by default. It can be optionally set to a different directory relative to CWD.',
```

Comment:
Per my other comment, this could change to:
```suggestion
  'build': 'Build a static web application (see https://storybook.js.org/docs)',
  'build_dir': 'Build output directory (defaults to examples/storybook)',
```

Also another nit / note: All task and flag descriptions are meant to fit on a single line and do not have a trailing period.

![image](https://user-images.githubusercontent.com/26553114/123149081-e75abe00-d42e-11eb-8f63-0fc07827de51.png)
![image](https://user-images.githubusercontent.com/26553114/123149126-f04b8f80-d42e-11eb-868c-b08a7d7a6e85.png)